lib/app/data.dart

```diff
@@ -47,7 +47,7 @@ class User implements Entity<User> {
           avatarInitials: data['avatarInitials'],
           avatarBackgroundColor:
               (data['avatarBackgroundColor'] as String).hexToColor,
-          permissions: (data['permissions'] as List<dynamic>).cast<String>(),
+          permissions: (data['permissions'] as List<dynamic>)?.cast<String>(),
```

Comment:
If there's no `permissions` field, shouldn't we just use an empty list? We treat this like the user has no permissions anyway.

Reply:
I'd prefer a `null` here to indicate we _don't know_ the actual permissions versus the user having no permissions.